packages/react-app/src/components/Account.jsx

```diff
@@ -50,6 +54,8 @@ export default function Account({
   loadWeb3Modal,
   logoutOfWeb3Modal,
   blockExplorer,
+  useBurner,
```

Comment:
we shouldn't name the variable `useBurner` as use* is reserved for hooks in react.

Reply:
okay we can rename that to just burner